2/README.md

```diff
@@ -54,29 +54,104 @@ There are X main use cases for Template:
+* The Device SHALL have one unique identifier which represents its Name
 
 ### Naming convention
+
 * The Device's Name SHALL use the following convention:
+
 ``` ABNF
 device-name = domain "/" family "/" member
 domain = 1*VCHAR
 family = 1*VCHAR
 member = 1*VCHAR
 ```
+
+ 
+### Device Interface
+
+The Device Interface is a list of Attributes, Pipes and Commands provided by a Device Instance.
+The Device instance SHALL expose (provides access to) all Attributes, Pipes and Commands defined by its Device Class.
+
+The Device instance MAY expose Attributes and/or Commands not defined by its Device Class. 
+These are called Dynamic Attributes and/or Dynamic Commands respectively, see [RFC-XX]().
+The Dynamic Attributes and Dynamic Commands MAY be added to the interface during Device Initialisation phase and/or Device Operation. 
+
+
+### Device lifecycle
+
+The Device lifecycle is:
+* Device Initialisation phase, when Device object is created and made visible in a Tango Controls system (see [RFC-1](), [RFC-8]())
+* Device Operation phase, when Device Interface MUST be available to actors of the Tango Controls system.
+* Device Destruction phase, when Device SHOULD be unregistered from the Tango Control system and Device Interface MAY not
+  be available within the Tango Controls system. 
+
+The device lifecycle MUST be implemented by a Device Server.
+
+### Device State
+
+Value of Device's State attribute MAY impact how the Device object respond to:
+
+* Writing to a Device's Attribute or to a Device's Pipe  
+* Calling a Device's Command
+
+Device MAY not execute (block) the attribute write or the pipe write operation or the command execution code for certain States.
+In such a case the the Device object SHALL respond with throwing an exception.  [TO-DO: specify which exception]
+The way how a Device object respond to the above calls define a Device State Machine.
+
+**NOTE**: Current implementation allows a Device object to block the operations irrespectively to the Device's State
+attribute value. 
+
+
+[TO-DO: Move the following to Device Server ? ]
+#### Device Initialization
+
+Device Initialisation phase is intended to prepare Device Object for Operation. 
+
+Device Initialization SHOULD be divided into two subsequent steps:
+
+1) Object creation, when an object is created in a space of a Device Server process. After this phase Device object 
+   SHALL be accessible locally and via network. During this phase the same `user code` which is executed when the 
+   Device Init command is executed MAY be called.
+
+2) Device exporting, when Device is made visible within the Tango Controls system. It SHALL use a Database System 
+   ([RFC-6]()) to provide information on how other actors in the system can access its interface. 
+
+
+The Device Server MUST implement at least Object creation step.
+
+#### Device Operation
+
+* Attribute read and write
+* Command Call
+* Device State handling
+
+#### Device Destruction
+
```

Comment:
@hardion, there is a similar question as for Device Class RFC. Should I create RFC-8 and move the text there?

Reply:
You meant Device Server. There is probably more to say about the Device Server since the all interface has to be defined.
You can simply add a Reserved Devices chapter where the domain "dserver" is reserved and point to the RFC/8.

Reply:
@piogor Do you think you can add this reserved domain "dserver" to the RFC this week?
You can take example to https://github.com/tango-controls/rfc/blob/draft/3/README.md#reserved-commands

Reply:
added